### PR TITLE
chore(mergify): check conventional commit on all PRs

### PR DESCRIPTION
**Pull request by DouglasBlackwood**
Check the conventional commit title on all PRs, except draft PRs.

### CODE REVIEW

**Review by lecrepont01 — APPROVED** [review]

File: .mergify.yml [hunk 1]
```diff
@@ -63,7 +63,7 @@ merge_protections:
   - name: Enforce conventional commit
     description: Make sure that we follow https://www.conventionalcommits.org/en/v1.0.0/
     if:
-      - base = main
+      - "-draft"
```
Comment (sileht) [inline .mergify.yml:66]: We use draft our self and we should have this check on our PR.
```suggestion
      - "-base~=^mergify/merge-queue/"
```

I'm unsure about the `/` excaping.

Comment (DouglasBlackwood) [reply .mergify.yml:66]: No sure it's necessary since MP won't run on MQ PRs.

Comment (sileht) [reply .mergify.yml:66]: Maybe we should just remove the `if`, WDYT?

Comment (DouglasBlackwood) [reply .mergify.yml:66]: Works for me.

**Review by Greesb — APPROVED** [review]

**Review by Syffe — APPROVED** [review]

**Review by sileht — APPROVED** [review]

**Comment by DouglasBlackwood** [comment]
@mergify queue

**Pull request merged**